providers/openfeature-provider-flagd/tests/e2e/flagd_container.py

```diff
@@ -11,11 +12,15 @@
 class FlagdContainer(DockerContainer):
     def __init__(
         self,
-        image: str = "ghcr.io/open-feature/flagd-testbed:v0.5.15",
+        image: str = "ghcr.io/open-feature/flagd-testbed",
         port: int = 8013,
         **kwargs,
     ) -> None:
-        super().__init__(image, **kwargs)
+        my_path = os.path.abspath(os.path.dirname(__file__))
+        path = os.path.join(my_path, "../../openfeature/test-harness/version.txt")
+        with open(path) as file:
+            data = file.read().rstrip()
```

Comment:
I recommend to use `Path` from `pathlib` here, it makes file handling much easier. There are couple of different ways to get to the path you want, so feel free to check it out
ex.
```suggestion
        path = Path(__file__).parents[2] / "openfeature/test-harness/version.txt"
        data = path.read_tex().rstrip()
```
you can also keep `../../` but then I recommend to invoke `.resolve()` once.

Reply:
uh nice, that is a way cleaner api - thank you